collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormsTask.java

```diff
@@ -240,6 +254,7 @@ private void cleanUp(FileResult fileResult, File fileOnCancel, String tempMediaP
             Timber.w("The user cancelled (or an exception happened) the download of a form at the "
                     + "very beginning.");
         } else {
+            formsDao.deleteFormsFromMd5Hash(new String[]{FileUtils.getMd5Hash(fileResult.file)});
```

Comment:
You can now simplify to
`formsDao.deleteFormsFromMd5Hash(FileUtils.getMd5Hash(fileResult.file));`